### PR TITLE
Expire tests and removed delete route

### PR DESCRIPTION
**Pull request by sktatsuno**

### CODE REVIEW

File: tests/integrationExpire.test.ts [hunk 1]
```diff
@@ -1,69 +1,77 @@
-import { assert } from 'https://deno.land/std@0.134.0/testing/asserts.ts';
+import { assert, assertEquals } from 'https://deno.land/std@0.134.0/testing/asserts.ts';
 import { superoak } from 'https://deno.land/x/superoak@4.7.0/mod.ts';
 import { connect } from 'https://deno.land/x/redis@v0.25.2/mod.ts';
 import { Application } from 'https://deno.land/x/oak@v10.2.0/mod.ts';
 import { Denostore } from '../mod.ts';
 import { typeDefs } from './schema/typeDefs.ts';
-import { resolvers } from './schema/resolver.ts';
+import { resolvers } from './schema/expireResolver.ts';
+import { delay } from "https://deno.land/std/async/mod.ts";
```
Comment (Jamesmjkim) [inline tests/integrationExpire.test.ts:8]: Could you add the version of the module for this import?

File: tests/integration.test.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { makeExecutableSchema } from 'https://deno.land/x/graphql_tools@0.0.2/mo
 // import { renderPlaygroundPage } from 'https://deno.land/x/oak_graphql@0.6.3/graphql-playground-html/render-playground-html.ts';
 import { Denostore } from '../mod.ts';
 import { typeDefs } from './schema/typeDefs.ts';
-import { resolvers } from './schema/resolver.ts';
+import { resolvers } from './schema/expireResolver.ts';
```
Comment (Jamesmjkim) [inline tests/integration.test.ts:13]: Is there a reason for changing this to the expireResolver?

**Review by Jamesmjkim — CHANGES_REQUESTED** [review]
I made some comments, take a look and let me know.

File: tests/integration.test.ts (hunk 2 above)
Comment (sktatsuno) [reply tests/integration.test.ts:13]: For the expire tests it's using a different resolver with expiration options

**Review by Jamesmjkim — APPROVED** [review]
looks good to me! Great work Scott!

**Pull request merged**